src/math/hyp1f1.hh

```diff
@@ -396,16 +400,18 @@ namespace rascal {
 
       double z_asympt{1.};
 
-      size_t nterms_a{0}, nterms_s{0};
+      // TODO(all) TODO(felix) I remember this was a quick fix and I am not
+      // sure if this should be added to master
+      // size_t nterms_a{0}, nterms_s{0};
       double h1f1_a{0}, h1f1_s{0};
       double z_above{0.}, z_below{0.};
 
       void update_switching_point() {
         this->h1f1_s = this->hyp1f1_series.calc(this->z_asympt);
         this->h1f1_a = this->hyp1f1_asymptotic.calc(this->z_asympt);
 
-        this->nterms_s = this->hyp1f1_series.n_terms;
-        this->nterms_a = this->hyp1f1_asymptotic.n_terms;
+        // this->nterms_s = this->hyp1f1_series.n_terms;
```

Comment:
away?

Reply:
don't understand this either, felix did this @felixmusil

Reply:
removed